pygsti/algorithms/germselection.py

```diff
@@ -4285,3 +4369,597 @@ def stable_pinv(mat):
     #new form the psuedoinverse:
     pinv= Vh.T@(pinv_s*U.T)
     return pinv
+
+
+#---------- Minimal Germ Spanning Vectors------------#
+#Not a great name for this section of code. Idea here is to
+#take a user-inputed AC germ-set (presumably from a prior germ
+#selection run and then identify a minimal set of amplified directions
+#that spans model space collectively accross all germs.
+#this can then be used as input to a version of FPR that is
+#globally aware of the overlap in amplified directions
+#of parameter space.
+
+def germ_set_spanning_vectors(target_model, germ_list, assume_real=False, float_type=_np.cdouble, 
```

Comment:
This appears to be defined here but only used in the FPR code. Thoughts on moving it into FPR?

Reply:
Good question. I bounced back and forth between putting this in the germ selection code or the FPR code before settling on germ selection for two main reasons. The first is that algorithmically this function shares much more in common with the algorithms used in germ selection, and uses a lot of the same helper functions which are defined in that file and are conversely not used in the fiducial pair reduction algorithms (so there'd be a lot of imports in the FPR code for functions only used in this context). The other was that in the long term I have applications in mind for this functionality that aren't directly related to fiducial pair reduction, so it was probably always going to have an imperfect home either way. I'm not deeply wedded to this though.

Reply:
Ok, that's good enough for me. Just wanted to make sure this was an intentional decision :)